packages/components/dialog/src/dialog.js

```diff
@@ -98,7 +91,7 @@ export class TSDialog extends TSElement {
 			if (this.notification === true && this.type === dialogTypes.SUCCESS && !this.nonDismissable) {
 				return window.setTimeout(() => {
 					this.dismissModal();
-				}, constants.delay.CLOSE);
+				}, 1500);
```

Comment:
🙈 Sorry, but can you please keep it as a constant but just in the component's utils/constants ?

Reply:
on it